shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h

```diff
@@ -15,7 +15,10 @@
 
 @interface DisplayLinkManager : NSObject
 
-@property(class, nonatomic, readonly) bool minimumFrameDurationDisabled;
+// Whether the max refresh rate on iPhone Pro-motion devices are enabled.
```

Comment:
Pro-motion iPad Pros too, right?

Reply:
This is only for iPhone, iPad always has high refresh rate on, let me add a comment about it too.